### DISCUSSION TITLE
Lesson7: test_can_pick_winner fails because of "i have no idea"

### DISCUSSION THREAD

**Discussion by sl00k** [smartcontractkit/full-blockchain-solidity-course-py#544] [Q&A] [answered]
This is the error I am getting when I only run "assert lottery.balance() == 0". Seems like no payout is happening
![image](https://user-images.githubusercontent.com/33859801/145718446-a3afa76d-8bf8-48f4-97b4-9d928f9ee1db.png)

This is what I get when I run "    assert lottery.recentWinner() == account"
![image](https://user-images.githubusercontent.com/33859801/145718543-4ed4e9b5-c934-4392-883f-f3556dc190ad.png)

And this is what I get when I delete the build fodler before testing
![image](https://user-images.githubusercontent.com/33859801/145718780-b280c235-6411-4e34-8c4b-661fee96d3bb.png)

**Comment by cromewar** [comment]
hello @sl00k the reason because `lottery.balance() == 0` don't pass is due to the account you are using, as the wallet which deploys the contract is the same as you pick as winner, the lottery balance will never be 0 because of this.

Regarding to the `invalid literal for int() with base = 16` this happens when yo try to use a integer passing a string instead, for example `int(' ', 16)` could you please share here the code from `Lottery.sol` and `helpful_scripts.py`?

**Comment by sl00k** [comment, marked as answer]
the code is the same as here in the repo.
changing the time.sleep helped here. (and for the rinkeby test chain as well)
![image](https://user-images.githubusercontent.com/33859801/145722906-cc822e6c-ab69-4b93-93f3-a8ea9e872576.png)

**Reply by cromewar** [reply]
Nice I'm glad you solved the problem.
